nevergrad/functions/powersystems/core.py

```diff
@@ -0,0 +1,233 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# This code is based on a code and ideas by Emmanuel Centeno and Antoine Moreau,
+# University Clermont Auvergne, CNRS, SIGMA Clermont, Institut Pascal
+
+import copy
+from math import sqrt, tan, pi
+import math
+import matplotlib.pyplot as plt
+from typing import Any
+from typing import List
+import numpy as np
+from nevergrad.common.typetools import ArrayLike
+from ... import instrumentation as inst
+from ...instrumentation.core import Variable
+from ...instrumentation.multivariables import Instrumentation
+pi = np.pi
```

Comment:
You are already doing from `math import sqrt, tan, pi` so you can remove this (unless you really need the pi from numpy instead of math, in that case do `from numpy import pi`)

Reply:
Ok!